src/auditlog/registry.py

```diff
@@ -92,7 +98,28 @@ def _connect_signals(self, model):
         """
         for signal in self._signals:
             receiver = self._signals[signal]
-            signal.connect(receiver, sender=model, dispatch_uid=self._dispatch_uid(signal, model))
+
+            if signal is m2m_changed:
+                # Connect Many to many signals
+                for m2m_field, m2m_signals in self._registry[model]['m2m_fields'].items():
+                    assert isinstance(m2m_signals, list)
+                    if m2m_signals == []:
+                        m2m_signals = 'post_add', 'post_remove', 'post_clear'
+
+                    field = getattr(model, m2m_field)
+                    m2m_model = getattr(field, 'through')
+                    if 'post_save' in m2m_signals:
+                        post_save_receiver = self._signals[post_save]
+                        post_save.connect(post_save_receiver, sender=m2m_model, dispatch_uid=self._dispatch_uid(m2m_model, model))
+
+                    if 'pre_save' in m2m_signals:
+                        pre_save_receiver = self._signals[pre_save]
+                        pre_save.connect(pre_save_receiver, sender=m2m_model, dispatch_uid=self._dispatch_uid(m2m_model, model))
+
+                    setattr(m2m_model, '_map_signals', m2m_signals)
+                    signal.connect(receiver, sender=m2m_model, dispatch_uid=self._dispatch_uid(signal, m2m_model))
```

Comment:
This `dispatch_uid` is unique per dependency, not per a pair of dependent + dependency. Consider this code:
```python
class Tag(Model):
    name = CharField(...)

class User(Model):
    random_tags = ManyToManyField(Tag)

class Cat(Model):
    color_tags = ManyToManyField(Tag)

auditlog.register(User, m2m_fields={"random_tags": []})
auditlog.register(Cat, m2m_fields={"color_tags": []})
```
In this case we won't track the changes to `Cat.color_tags`, because its handler will have the same `dispatch_uid` as the one for `User.random_tags` (please correct me if I misunderstand the logic of `dispatch_uid`).

A better way would be to generate `dispatch_uid` from a pair of `User, Tag` and `Cat, Tag` respectively.